.parcelrc

```diff
@@ -9,7 +9,6 @@
   },
   "namers": ["parcel-namer-docs", "..."],
   "packagers": {
-    "*.json": "parcel-packager-docs",
```

Comment:
i had to remove this because it was expecting a default export (and i moved to *not* have one), so we will have to modify.